apps/ejabberd/test/xmpp_route_SUITE.erl

```diff
@@ -2,64 +2,92 @@
 -compile([export_all]).
 
 -include_lib("exml/include/exml.hrl").
+-include_lib("eunit/include/eunit.hrl").
 
 
 all() ->
-    [ success_with_module_implementing_behaviour,
-      fail_with_module_not_implementing_behaviour,
-      fail_when_do_route_crashes
-    ].
+    [ basic_routing ].
 
 init_per_suite(C) ->
-    application:start(stringprep),
+    application:ensure_started(stringprep),
```

Comment:
It might be a bit safer to use `ensure_all_started/1`. It does the same and even more (more: starts dependencies recursively) and doesn't require specific knowledge (does the app have any dependencies?) about the particular application passed as the argument. I'm nitpicking, yes - it's not a serious problem here.